### PR TITLE
Option fileonly for noah to only store tweets in a file, not geotag nor ingest

### PR DESCRIPTION
**Pull request by MoniMoledo**

### CODE REVIEW

**Comment by chenlica** [comment]
Glad to see your first PR!

**Comment by JavierJia** [comment]
@MoniMoledo you should run 
`git pull` to update your local branch.

File: noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/TwitterFeedStreamDriver.java [hunk 1]
```diff
@@ -78,36 +78,40 @@ public void run(Config config)
 
         // Establish a connection
         try {
-            twitterClient.connect();
+           twitterClient.connect();
```
Comment (JavierJia) [inline noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/TwitterFeedStreamDriver.java]: did you reformat the code?

**Comment by JavierJia** [comment]
LGTM. Nicely done! 👍

**Pull request merged**